File: scripts/config_loader.py
import os
import json

def load_config():
    # 获取脚本所在目录的父目录（项目根目录）
    base_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
    config_path = os.path.join(base_dir, 'config.json')
    
    with open(config_path, 'r') as config_file:
        config = json.load(config_file)
    
    # 将相对路径转换为绝对路径
    config['pdfDir'] = os.path.abspath(os.path.join(base_dir, config['pdfDir']))
    
    return config

config = load_config()

File: src/configLoader.js
const fs = require('fs');
const path = require('path');

const loadConfig = () => {
  const configPath = path.join(__dirname, '..', 'config.json');
  const rawConfig = fs.readFileSync(configPath, 'utf8');
  const config = JSON.parse(rawConfig);
  
  // 转换相对路径为绝对路径
  config.pdfDir = path.join(__dirname, '..', config.pdfDir);
  
  return config;
};

module.exports = loadConfig();